std_msgs/Header header
geometry_msgs/Pose pose

int32 cloud_size
vo/PointID_msg[] cloud